filter_plugins/host_addresses.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
That's is definitely why I'm getting errors I'm getting. OK, I agree that using built-ins is better. Will change, thanks.